examples/build.provenance

```diff
@@ -0,0 +1,205 @@
+{
+    "_type": "https://in-toto.io/statement/v0.1",
+    "subject": [
+      {
+        "Name": "salsa.txt",
+        "Digest": {
+          "sha256": "f8161d035cdf328c7bb124fce192cb90b603f34ca78d73e33b736b4f6bddf993"
+        }
+      }
+    ],
+    "predicateType": "https://in-toto.io/provenance/v0.1",
+    "predicate": {
+      "builder": {
+        "id": "https://github.com/Attestations/GitHubHostedActions@v1"
+      },
+      "metadata": {
+        "buildInvocationId": "936659519",
+        "completeness": {
+          "arguments": true,
+          "environment": true,
+          "materials": false
+        },
+        "reproducible": false,
+        "buildFinishedOn": "2021-06-14T17:45:22Z"
+      },
+      "recipe": {
+        "type": "https://github.com/Attestations/GitHubActionsWorkflow@v1",
+        "definedInMaterial": 0,
+        "entryPoint": "Create a sample provenance",
+        "arguments": null,
+        "environment": {
```

Comment:
at least according to [this example](https://github.com/in-toto/attestation/blob/main/spec/predicates/provenance.md#github-actions) this might have a lot more information than needed.  Maybe that's fine?  It's probably a lot easier than reducing what's reported.